auth-service/src/main/resources/templates/thymeleaf/emails/no-user-account.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
```

Comment:
We should make the header and footer fragments like the page HTML templates

Reply:
Is what I've now done what you meant?

Reply:
I think put the fragments in a separate file, maybe inside the email folder, but yes, exactly that 🙂